example/main.go

```diff
@@ -23,9 +23,9 @@ var (
 )
 
 func main() {
-	root.ParseArgs()
+	_= root.Parse()
```

Comment:
Why?

Reply:
I changed `ParseArgs` to `Parse`, since `ParseArgs` would immediatly trigger the `panic` we discussed earlier. `Parse` on the other hand will succeed and will pass whatever `os.Args` holds to the command, which is at least the program name.

The `_ =` part was added to explicitly ignore the returned error. I prefer doing this to clearly state my indent that I don't want to deal with the return value, instead of leaving this as an excercise to the reader :D. I can remove this if you don't agree, though :).

Reply:
Oh, right, this is a mistake! Good catch!
I think I prefer to just remove the `_=`, since it might confuse users...
If you insist, just add a space between the underscore and the equal sign.

Reply:
Fair enough, I remove the `_ =` :). Thanks for being so patient with me! 👍